### ISSUE TITLE
Delete this Repo

### ISSUE THREAD

**Issue by topowright-zz** [ArcGIS/military-tools-webappbuilder#115]
I do not think we need this repo anymore.

**Comment by topowright-zz** [comment]
I would like this to be deleted. All content has been moved to the Solutions widget Repo. @nfeuerstein @lfunkhouser

**Comment by nfeuerstein** [comment]
Please do!

**Comment by lfunkhouser** [comment]
Agreed. I have added the THIS_IS_NO_LONGER_MAINTAINED.md file and updated the README.md. Because this is a public repository that has been forked, we will wait one release cycle to delete.
